tests/cluster_tests/test_pbspro.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
```

Comment:
Is this file a copy or similar to our existing test script file?  Either way it may be nice to have a commented section that details what is going on in this file and possibly when or how it should be amended (in the case where a new developer needs to add a test case).